packages/webview_flutter/pubspec.yaml

```diff
@@ -25,3 +25,5 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
+  mockito: ^5.0.16
+  pedantic: ^1.10.0
```

Comment:
In case ``mockito: ^5.0.16`` , I got the following error without that:
```
  error • test/webview_flutter_test.dart:11:8 • Target of URI doesn't exist: 'package:mockito/annotations.dart'. Try creating the file referenced by the URI, or Try using a URI for a file that does
          exist. • uri_does_not_exist
  error • test/webview_flutter_test.dart:12:8 • Target of URI doesn't exist: 'package:mockito/mockito.dart'. Try creating the file referenced by the URI, or Try using a URI for a file that does exist.
          • uri_does_not_exist
  error • test/webview_flutter_test.dart:20:1 • Undefined name 'GenerateMocks' used as an annotation. Try defining the name or importing it from another library. • undefined_annotation
  error • test/webview_flutter_test.dart:32:5 • The function 'when' isn't defined. Try importing the library that defines 'when', correcting the name to the name of an existing function, or defining a
          function named 'when'. • undefined_function
  error • test/webview_flutter_test.dart:33:16 • The argument type 'dynamic' can't be assigned to the parameter type 'BuildContext?'. • argument_type_not_assignable
  error • test/webview_flutter_test.dart:33:16 • The function 'anyNamed' isn't defined. Try importing the library that defines 'anyNamed', correcting the name to the name of an existing function, or
          defining a function named 'anyNamed'. • undefined_function
...
``pedantic: ^1.10.0`` will be removed.

Reply:
How did you get the error? There's no such file (`test/webview_flutter_test.dart`) in this repository.

Reply:
I'm sorry 😢 I'm missing some files. I'll include it in the next commit.

Reply:
Our plugins usually don't have unit tests or widget tests and our CI hasn't been set up for such tests. Are you sure you want to add the test? 😮

Reply:
> Are you sure you want to add the test?

After hearing this, It may not be necessary right now. 😄 
I do this from time to time.. I often get confused about where to bring it..